internal/injector/builtin/yaml/databases/mongo.yml

```
@@ -0,0 +1,25 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2023-present Datadog, Inc.
+%YAML 1.1
+---
+meta:
+  name: go.mongodb.org/mongo-driver/mongo
+  description: Official Golang driver for MongoDB.
+  icon: database
+aspects:
+  - id: Inject mongotrace.NewMonitor
+    join-point:
+      one-of:
+        - function-call: go.mongodb.org/mongo-driver/mongo.Connect
```

No need to `one-of` if there is only one option :)

```suggestion
    join-point:
      function-call: go.mongodb.org/mongo-driver/mongo.Connect
```